h2o-py/tests/testdir_algos/automl/pyunit_automl_get_automl.py

```diff
@@ -37,5 +37,7 @@ def test_get_automl():
     predictions_from_output = get_aml.predict(train)
     assert (predictions == predictions_from_output).all()
 
+    # Test get_leaderboard PUBDEV-7454
+    assert (get_leaderboard(aml) == get_leaderboard(get_aml)).all()
```

Comment:
can you please also double check that it works with the extra columns? It should, but better verify:
```get_leaderboard(get_aml, 'ALL')```